kong/runloop/plugin_servers/init.lua

```diff
@@ -15,14 +15,25 @@ local kong = kong
 local ngx_var = ngx.var
 local ngx_sleep = ngx.sleep
 local worker_id = ngx.worker.id
+local req_start_time = ngx.req.start_time
+
 local coroutine_running = coroutine.running
 local get_plugin_info = proc_mgmt.get_plugin_info
 local get_ctx_table = require("resty.core.ctx").get_ctx_table
-local subsystem = ngx.config.subsystem
+
+local is_http = ngx.config.subsystem == "http"
 
 local cjson_encode = cjson.encode
 local native_timer_at = _G.native_timer_at or ngx.timer.at
 
+local req_get_headers
+local resp_get_headers
+
+if is_http then
+  req_get_headers  = ngx.req.get_headers
+  resp_get_headers = ngx.resp.get_headers
+end
```

Comment:
https://github.com/openresty/lua-nginx-module#ngxreqget_headers
https://github.com/openresty/lua-nginx-module#ngxrespget_headers

These two functions are not available in the `init` phase, I think we can't localize them.

Reply:
I created a small nginx config file, wrote some code in `init_by_lua_block`:

```
    init_by_lua_block {
        local x = ngx.req.get_headers
        ngx.log(ngx.ERR, "type = ", type(x))
    }
```

It works. So I think the reason is that the module is loaded in stream subsystem,
and there is no `ngx.resp.xxx`.